[FEATURE] Ajout du code de vérification dans le certificat (PIX-437)

Anne-Gaelle-S opened the pull request:
## :unicorn: Problème
Dans le cadre de la vérification d'un certificat, nous souhaitons afficher un code de vérification sur le certificat des utilisateur de app.pix.fr. Ce code pourra ensuite être copié puis envoyé à une tierce personne afin qu'elle puisse consulter / vérifier l'authenticité du certificat de l'utilisateur.
Aujourd'hui nous générons le code de vérification (composé de 6 caractères random) lors du lancement du test de certification d'un utilisateur. (Il n'y a pas encore de code pour les personnes ayant déjà passé leur certification)
Le formulaire où l'on peut entrer le code afin de vérifier un certificat est en cours.

## :robot: Solution
Cette PR vise à afficher le code de vérification (s'il existe pour l'utilisateur).

Résultat : 
<img width="1070" alt="image" src="https://user-images.githubusercontent.com/38167520/91426261-8e48ea80-e85c-11ea-90c2-2438f5e3655b.png">


## :rainbow: Remarques
Cette feature est soumise à un toggle (elle sera donc désactivée et invisible pour tout le monde jusqu'à qu'il soit décidé de mettre la feature visible).
La variable d'environnement est : `FT_IS_SHARED_CERTIFICATE_ACTIVE`.

## :100: Pour tester
Mettre le FT en mode "on" : 
- `FT_IS_SHARED_CERTIFICATE_ACTIVE=true npm start` 
- Aller sur la page de certificat d'un utilisateur (par ex: `certif-success@example.net`)
- Constater qu'il y a un encadré supplémentaire avec un code de vérification dans le header.
- Constater qu'au clic de l'icon bleue le code est bien copié dans le presse papier

Mettre le FT en mode "off": 
- `FT_IS_SHARED_CERTIFICATE_ACTIVE=false npm start` ou juste `npm start`
- Constater que sur la page de certificat (même user) il n'y a plus l'encadré de verification-code

Anne-Gaelle-S committed:
[API] Add verificationCode in test tooling

Anne-Gaelle-S committed:
[API] Add verificationCode in serializer

Anne-Gaelle-S committed:
[API] BSR add more detail on GET certifs route

pix-service commented:
I'm deploying this PR to these urls:

- App: https://app-pr1791.review.pix.fr
- Orga: https://orga-pr1791.review.pix.fr
- Certif: https://certif-pr1791.review.pix.fr
- Admin: https://admin-pr1791.review.pix.fr
- API: https://api-pr1791.review.pix.fr/api/

 Please check it out!

Anne-Gaelle-S committed:
[Mon-Pix] Add verificationCode in model

Anne-Gaelle-S committed:
[Mon-Pix] Show verificationCode on certificate

Anne-Gaelle-S committed:
[Mon-Pix] BSR certif design review with Quentin

Anne-Gaelle-S committed:
[Mon-Pix] i18n verification-code

Anne-Gaelle-S committed:
[Mon-Pix] Add feature toggle to show verificationCode

Anne-Gaelle-S committed:
[Mon-Pix] Update and install dependencies

Pix-UI and font-awesome were updated
Clip-board was install (to use CopyButton component)

Anne-Gaelle-S committed:
[Mon-Pix] Add copy icon to config

aceol commented on api/lib/domain/models/PrivateCertificate.js:
```diff
@@ -15,6 +15,7 @@ class PrivateCertificate {
     commentForCandidate,
     cleaCertificationStatus,
     resultCompetenceTree = null,
+    verificationCode,
```
Ici le verificationCode est ajouté mais a deja ete introduit dans les tests sur le commit precedent. Est-ce qu'on peut merger les deux pour plus de clarté?
